commands/InjectedCommand.java

```diff
@@ -1,58 +1,45 @@
 package org.usfirst.frc4904.standard.commands;
 
+import edu.wpi.first.wpilibj2.command.CommandBase;
 
-import edu.wpi.first.wpilibj.command.Command;
+public abstract class InjectedCommand extends CommandBase {
+	private final CommandBase previous;
 
-public abstract class InjectedCommand extends Command {
-	private final Command previous;
-	
-	public InjectedCommand(Command previous) {
+	public InjectedCommand(String name, CommandBase previous) {
 		super();
+		setName(name);
 		this.previous = previous;
 	}
-	
-	public InjectedCommand(String name, Command previous) {
-		super(name);
-		this.previous = previous;
-	}
-	
-	public InjectedCommand(double timeout, Command previous) {
-		super(timeout);
-		this.previous = previous;
-	}
-	
-	public InjectedCommand(String name, double timeout, Command previous) {
-		super(name, timeout);
-		this.previous = previous;
+
+	public InjectedCommand(CommandBase previous) {
+		this("Injected(" + previous.getName() + ")", previous);
 	}
-	
+
 	@Override
-	final protected void initialize() {
-		if (previous != null && (previous.isRunning() || !previous.isCanceled())) {
+	final public void initialize() {
```

Comment:
`public final` not `final public`

Reply:
@Exr0n did you forget to push?

Reply:
I think my changes got consumed somewhere because I remember editing this? Should be fixed now though.